### DISCUSSION TITLE
[6] Further progress

### DISCUSSION THREAD

**Discussion by GermanJesus-lul** [GermanJesus-lul/Rezeptbuch#7] [Announcements]
Hey,

This week, we made some progress by setting up the initial GUI for our application!
Although it’s still a work in progress, we now have a simple example: a landing page that leads to a list of recipes.
The design and functionality are far from complete, but it's a solid starting point
The design and functionality are not nearly finished.  

![image](https://github.com/user-attachments/assets/6bc2c689-fea6-40b7-afb7-ba0d6ff2da1a)  
![image](https://github.com/user-attachments/assets/daa1a729-84de-45d9-8498-db6caecfc39a)  

On the backend side, we’ve already implemented the logic for loading a list of recipes from the server. We followed a Test-Driven Development (TDD) approach for this, so we already have tests in place to ensure our code is reliable and well-structured.  

![image](https://github.com/user-attachments/assets/bdda63db-2e17-451c-806e-0c289d169b53)

Yours truly,
The Rezeptbuch team

**Comment by AlinaBoess** [comment]
Hey,  

great progress! 😊 A first GUI and the TDD-based backend logic sound very promising. Starting with a landing page and recipe list is a great way to kick things off – keep it up, Rezeptbuch-Team! 🚀  

Best regards,  
Das Restaurantreservierungsteam
